databases/redis/src/opnsense/mvc/app/models/OPNsense/Redis/Redis.xml

```diff
@@ -16,7 +17,7 @@
         <Required>Y</Required>
       </protected_mode>
       <port type="IntegerField">
-        <MinimumValue>1</MinimumValue>
+        <MinimumValue>0</MinimumValue>
         <MaximumValue>65536</MaximumValue>
```

Comment:
if you change that, could you be so nice to decrease the maximum by 1 as well - it is currently wrong.